notes/cli/inspect.sh

```diff
@@ -0,0 +1,194 @@
+sqlite3 $BASEDIR/db.sqlite ".schema"
+
+poetry run spinta inspect $BASEDIR/manifest.csv -o $BASEDIR/manifest.csv
+#| Loading InlineManifest manifest default
+#| Loading InlineManifest manifest default
+
+poetry run spinta show $BASEDIR/manifest.csv
+#| id | d | r | b | m | property   | type    | ref     | source                                        | prepare | level | access | uri | title | description
+#|    | datasets/examplle          |         |         |                                               |         |       |        |     |       |
+#|    |   | db                     | sql     |         | sqlite:///var/instances/cli/inspect/db.sqlite |         |       |        |     |       |
+#|    |                            |         |         |                                               |         |       |        |     |       |
+#|    |   |   |   | City           |         | id      | cities                                        |         |       |        |     |       |
+#|    |   |   |   |   | id         | integer |         | id                                            |         |       |        |     |       |
+#|    |   |   |   |   | name       | string  |         | name                                          |         |       |        |     |       |
+#|    |   |   |   |   | country    | ref     | Country | country_id                                    |         |       |        |     |       |
+#|    |                            |         |         |                                               |         |       |        |     |       |
+#|    |   |   |   | Country        |         | id      | countries                                     |         |       |        |     |       |
+#|    |   |   |   |   | id         | integer |         | id                                            |         |       |        |     |       |
+#|    |   |   |   |   | name       | string  |         | name                                          |         |       |        |     |       |
+#|    |   |   |   |   | continent  | integer |         | continent_id                                  |         |       |        |     |       |
+#|    |   |   |   |   | population | integer |         | population                                    |         |       |        |     |       |
+#|    |                            |         |         |                                               |         |       |        |     |       |
+#|    |   |   |   | Continent      |         | id      | continents                                    |         |       |        |     |       |
+#|    |   |   |   |   | id         | integer |         | id                                            |         |       |        |     |       |
+#|    |   |   |   |   | name       | string  |         | name                                          |         |       |        |     |       |
+# FIXME: `continent` type was lost, types specified in manifest table should be preserved.
```

Comment:
Property types specified in manifest file, should be preserved even if type in data source has changed. For example here, foreign key was not specified in data source, but it was specified in manifest file, so after updating manifest from source, we should preserve foreign key in manifest file.